package.json

```diff
@@ -65,7 +65,7 @@
     "classnames": "^2.3.1",
     "date-fns": "^2.29.2",
     "ethereum-blockies-base64": "^1.0.2",
-    "ethers": "^5.6.8",
+    "ethers": "5.7.2",
```

Comment:
the core sdk imports type from ethers and it has a constant string type on `version` variable equal to 5.7.2, hence the pinning